dir_to_study/transform.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+import unicodedata
+import uuid
+from datetime import timezone, datetime
+import hashlib
+
+import magic
+import pathlib
+import click
+import logging
+
+import orjson
+import mimetypes
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+ACED_NAMESPACE = uuid.uuid3(uuid.NAMESPACE_DNS, 'aced-ipd.org')
+
+
+def create_research_study(name, description):
+    """Creates bare-bones study."""
+    study = {
+        'title': name,
+        'id': str(uuid.uuid5(ACED_NAMESPACE, name)),
+        'description': description,
+        'status': 'active',
+        "resourceType": "ResearchStudy",
+    }
+    return study
+
+
+# open file pointers
+emitters = {}
+
+
+def emitter(output_path: pathlib.Path, name: str):
+    """Maintain a hash of open files."""
+    if name not in emitters:
+        emitters[name] = open(output_path / f"{name}.ndjson", "wb")
+    return emitters[name]
+
+
+def md5sum(file_name):
+    """Calculate the hash and size."""
+    md5_hash = hashlib.md5()
+    file_name = unicodedata.normalize("NFKD", str(file_name))
+    with open(file_name, "rb") as f:
+        # Read and update hash in chunks of 4K
+        for byte_block in iter(lambda: f.read(4096), b""):
+            md5_hash.update(byte_block)
+
+    return md5_hash.hexdigest()
+
+
+def _extract_fhir_resources(file):
+    """A placeholder to parse other resources from file."""
+    pass
+
+
+@click.command('create')
+@click.option('--project_id', required=True,
+              default=None,
+              show_default=True,
+              help='Gen3 program-project'
+              )
+@click.option('--input_path', required=True,
+              default=None,
+              show_default=True,
+              help='Read files from this path'
+              )
+@click.option('--output_path', required=True,
+              default=None,
+              show_default=True,
+              help='Write FHIR resources to this path'
+              )
+@click.option('--pattern',
+              default='**/*',
+              show_default=True,
+              help='File names to match.')
+def _transform(project_id, input_path, output_path, pattern):
```

Comment:
Since you're using _transform as the driver function, I had to change the file name to _transform.py to get:  'python dir_to_study/_transform.py --project_id aced-test --input_path ./tests/fixtures/ --output_path test'
command to work for me.

Reply:
Thanks, resolved below.